src/dbnode/storage/limits/query_limits.go

```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	"github.com/m3db/m3/src/x/instrument"
+
```

Comment:
is this intentional?

Reply:
Yes unfortunately the linter is temporarily not running... I've asked a few folks to look into reviving the linter (well.. use a new linter, metalinter we used previously is dead) but no progress yet.

Reply:
We order imports:
1. stdlib
2. imports from this repo
3. all other repos

Reply:
i'm guessing we want to enable https://github.com/golangci/golangci-lint ?